tests/integration/README.md

```diff
@@ -0,0 +1,33 @@
+## Running Integration Tests with `aws-k8s-tester`
+
+[`aws-k8s-tester`](https://github.com/aws/aws-k8s-tester) is an e2e test deployment interface for testing K8s clusters on AWS and is used in conjunction with `aws-ebs-csi-driver` for csi integration tests.
+
+### Requirements 
+
+1. macOS or Linux
+1. [Go](https://golang.org/doc/install), version 1.11+
+1. `GOPATH` environment variable [set](https://github.com/golang/go/wiki/SettingGOPATH)
+1. [AWS account](https://aws.amazon.com/account/) that has been [configured locally](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html)
+1. download `aws-k8s-tester`: `go get github.com/aws/aws-k8s-tester`
```

Comment:
Since we are downloading for user already, no need to mention download since this will just be implementation details.

Reply:
done!